main.tf

```
@@ -1,20 +1,22 @@
 terraform {
   required_providers {
     spotify = {
-      version = "~> 0.1.5"
+      version = "~> 0.2.6"
       source  = "conradludgate/spotify"
     }
   }
 }
 
-variable "spotify_api_key" {
-  type = string
-}
-
 provider "spotify" {
   api_key = var.spotify_api_key
 }
 
+data "spotify_search_track" "by_artist" {
+  artists = ["Dolly Parton"]
```

With provider version 0.2.6 this won't work anymore:
```
│ Error: Unsupported argument
│
│   on main.tf line 30, in data "spotify_search_track" "by_artist":
│   30:   artists = ["Gareth Emery"]
│
│ An argument named "artists" is not expected here. Did you mean "artist"?
```
Therefore it should be:
```suggestion
  artist = "Dolly Parton"
```

---

Doh! I did that locally but forgot

---

No worries, thanks for fixing in the first place